Gemfile

```diff
@@ -7,11 +7,7 @@ ruby '~> 2.6.3'
 gem 'rails', '~> 6.0.2'
 # Use Puma as the app server
 gem 'puma', '~> 4.3'
-# Use SCSS for stylesheets
-gem 'sass-rails', '>= 6'
```

Comment:
I don't think you need to remove this. We are going to have CSS via the asset pipeline so we might want to have SASS.

If you create a new rails 6 app you'll see this gem is present along with webpacker gem